coscreen docs updates (#20187)
DataDog/documentation: content/en/coscreen/troubleshooting.md

is_beta: false
---

### Why does the audio quality degrade when using the microphone of my Bluetooth headset as input in CoScreen, Zoom, and other tools?

If you're using a Bluetooth headset, the playback quality may degrade when your headset's microphone is selected as an audio input device. You may notice this if you play audio (for example, play a YouTube video) while you are in a CoScreen session. This can occur because your Bluetooth headset has switched to using a different Bluetooth profile.

> When only playing audio, Bluetooth headsets typically use the [A2DP profile][2], which is optimized for high audio quality but does not support using the microphone. If you choose your headset microphone as audio input (for example, during a CoScreen session or Zoom meeting) the headset switches to a different profile, usually [HFP][3] or [HSP][4], which supports microphone usage but has lower sound quality. Most Bluetooth headsets can use only one profile at a time.

Reviewer: ```suggestion
When only playing audio, Bluetooth headsets typically use the [A2DP profile][2], which is optimized for high quality audio but does not support using the microphone. When selecting your headset microphone as input, (for example, in CoScreen or Zoom), it switches to the [HFP][3] or [HSP][4] profile with lower audio quality. Most Bluetooth headsets can use only one profile at a time.
```
My attempt at appeasing the linter and making this sentence 25 words.

> To avoid this issue, you can use a different audio input—such as a laptop's built-in microphone. You may need to restart your application to regain high audio quality.

Reviewer: ```suggestion
To avoid this issue, you can use a different audio input—such as a laptop's built-in microphone. You may need to restart your application to regain high  quality audio.
```
Both ways are correct, but this sounds more natural to my ears (eyes).
